js/src/redux/reducers/openSidebarSections.js

```diff
@@ -0,0 +1,49 @@
+import { CLOSE_ALL_SIDEBAR_SECTIONS, OPEN_SIDEBAR_SECTION, CLOSE_SIDEBAR_SECTION } from "../actions/openSidebarSections";
```

Comment:
If you write functions to handle state change I think it's better to name them after the action instead of their responsibility. So I think `sidebarSectionOpener` should just be `openSidebarSection` for example.

Reply:
I agree, but for readability's sake I used `sidebarSectionOpener()` because of overlap with the redux action `OPEN_SIDEBAR_SECTION` and its creator `openSidebarSection()`. 

Will consider changing.